### PR TITLE
fix(sematext sink): Rename cloud and add host

### PR DESCRIPTION
**Pull request by LucioFranco**
Signed-off-by: Lucio Franco <luciofranco14@gmail.com>

### CODE REVIEW

**Comment by LucioFranco** [comment]
@megastef sorry for the delay on this one, here are the changes following your comment. Let me know if there are any other changes you would like :)

**Comment by binarylogic** [comment]
Don't we need changes to the options in the TOML file?

**Comment by LucioFranco** [comment]
@binarylogic yeah, pushing that now :+1:

**Comment by Hoverbear** [comment]
Related to https://github.com/timberio/vector/issues/1000

File: .meta/sinks/sematext.toml [hunk 1]
```diff
@@ -19,11 +19,17 @@ request_in_flight_limit = 5
...
-examples = ["north_america", "europe"]
-description = "The cloud destination to send logs to."
+required = false
+examples = ["na", "eu"]
+description = "The region destination to send logs to. This option is required if `host` is not set."
+
+[sinks.sematext.options.host]
+type = "string"
+required = false
+examples = ["http://8.8.8.8", "http://example.com"]
```
Comment (Hoverbear) [inline .meta/sinks/sematext.toml]: Please don't use Google's IP as an example, use something from `127.x.x.x` or `10.x.x.x` :)

File: config/vector.spec.toml [hunk 2]
```diff
@@ -4605,6 +4598,24 @@ end
   healthcheck = true
   healthcheck = false
 
+  # The host that will be used to send logs to. This option is required if
+  # `region` is not set.
+  #
+  # * optional
+  # * no default
+  # * type: string
+  host = "http://8.8.8.8"
```
Comment (Hoverbear) [inline config/vector.spec.toml]: Please don't use Google's IP as an example, use something from 127.x.x.x or 10.x.x.x :)

File: website/docs/reference/sinks/sematext.md [hunk 3]
```diff
@@ -60,11 +59,12 @@ import CodeHeader from '@site/src/components/CodeHeader';
   # REQUIRED - General
   type = "sematext" # must be: "sematext"
   inputs = ["my-source-id"] # example
-  cloud = "north_america" # example
   token = "some-sematext-token" # example
 
   # OPTIONAL - General
   healthcheck = true # default
+  host = "http://8.8.8.8" # example, no default
```
Comment (Hoverbear) [inline website/docs/reference/sinks/sematext.md]: Please don't use Google's IP as an example, use something from 127.x.x.x or 10.x.x.x :)

**Comment by Hoverbear** [comment]
This appears to fix the points listed in #1000

**Pull request merged**